src/DynamoCore/Configuration/PreferenceSettings.cs

```diff
@@ -753,7 +755,7 @@ public static PreferenceSettings Load(string filePath)
             {
                 if (settings == null)
                 {
-                    return new PreferenceSettings();
+                    return new PreferenceSettings() { isValid = false };
```

Comment:
this seems a little funny - if we do return a brand new `PreferenceSettings` object, it should be a valid preferences - just not the one that the user tried to load... maybe the bool needs a different name.

Reply:
Rename works, done.